engine/lua/cloe/schedule.lua

```diff
@@ -370,5 +383,39 @@ function cloe.test_fixture(test)
         cloe.scheduler.execute_action(action)
     end
 
+    local function count_leading_tabs(str)
```

Comment:
This is a generic utility function that doesn't need access to any of the variables inside this function, so we should move it to the outer scope.